src/macro-carousel/macro-carousel.js

```diff
@@ -1145,15 +1145,15 @@ value. Add CSS units to its value to avoid breaking the slides layout.`);
     let isSlideInView;
     this._slides.map(slide => slide.element).forEach((slideEl, slideIndex) => {
       isSlideInView = !isUndefined(slidesInView.find(i => i === slideIndex));
-      // Slides in view have `aria-hidden` set to `false`.
+
+      // Ensure DOM is not accessible unless it is in the view.
+      slideEl.inert = !isSlideInView;
       slideEl.setAttribute(ATTRS.STANDARD.ARIA.HIDDEN,
           isSlideInView ? ATTR_VALUES.FALSE : ATTR_VALUES.TRUE);
+
       // Slides in view don't have the `inert` attribute and can be focused.
       if (isSlideInView) {
-        slideEl.removeAttribute(ATTRS.STANDARD.INERT);
-        slideEl.setAttribute(ATTRS.STANDARD.TABINDEX, -1);
-      } else {
-        slideEl.setAttribute(ATTRS.STANDARD.INERT, '');
+        slideEl.setAttribute(ATTRS.STANDARD.TABINDEX, '-1');
```

Comment:
Judging at the comment just before the `if`, it looks like I left a bug in those lines.
I think, if `isSlideInView` is `true`, the `tabindex` attribute should be removed (instead of being set to `-1`)

Reply:
I think its currently working as intended, we want to give `tabindex -1` so we can programmatically focus the slide right?

[https://mzl.la/2LY933V](https://mzl.la/2LY933V)
<img width="540" alt="Screenshot 2019-10-08 at 18 01 48" src="https://user-images.githubusercontent.com/1285982/66416486-d0bfc980-e9f5-11e9-9d25-63e9d66e28d9.png">

Reply:
`wicg-inert` seems to do some magic with tabindex too.. 😕